Airports/EGJB/Ownership.txt

```diff
@@ -1,31 +1,31 @@
 SECTOR:Only JBGND 1:0:0
-OWNER:JBR:JBA:JBT:JBG
+OWNER:JBR:JBF:JBT:JBG
```

Comment:
In position.txt, you are changing JBA to JBR, and JBR to JBF, here (and in the rest of this file) you're changing JBA to JBF and leaving JBR - presumably this changes the ownership order. Can I just check that is intentional? From the issue I thought you were just changing the identifiers round, not the actual ownership order, unless I misunderstood.

Ta!

Reply:
@hazzas-99 Can I seek your advice on this? I think I need to check my own understanding. I've never been too sure of the ordering on the "Only _____" ownership.

Reply:
Thank you!

@PLM1995 Since JBA no longer exists I've had to replace it with JBF in these instances. So if JBR goes offline, JBF takes ownership of the APP "Sector" (unlikely scenario but could happen if JBR disconnects before JBF). Forgot to do it with JA and as a result the compiler failed, and rightly so.

Reply:
OK I'm royally confused.

JBA has been changed to JBR not JBF in `positions.txt`, so changing the instances of JBA to JBF and leaving JBR instances the same in this file will change ownership order surely?

I certainly think I'm seeing things change with this PR vs. current version when simulating having EGJB_F_APP and EGJB_APP online. Whether it's the `Only _____` stuff that's doing it or the changes further down the file (lines 17-20 and 26-29) I don't know. It might be it was wrong before and this is fixing something but I don't think it's still the same order as you're effectively swapping EGJB_F_APP and EGJB_APP.

Sorry if I didn't explain that very well, basically I get different red circles before and after applying this PR in my testing setup, see below (EGJB_APP's active sector backgrounds in red):

----

Current Behaviour:

Nothing else online:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/7765ee37-da03-4937-98e5-a9765c9e0e7b)

With EGJB_F_APP also online:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/ecb1f8b6-b466-4a27-9f0c-afc188a6fbd1)

----

New Behaviour:

WIth EGJB_F_APP online:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/634a48d1-0092-48c5-9702-8f7da597dce1)

Reply:
That's how it should work as we don't have a "FIN" sector defined, so the ownership for the "APP" sector will remain with EGJB_APP, even when EGJB_F_APP is online.

Whether you want to define a new sector for EGJB_F_APP is up to you guys but for the amount of times we use it (ie. never) I'm not sure if it's worth it

Reply:
The idea is that JBF comes later in the order as they are in effect the FIN controller in this case. So whatever I have changed, because it looks like I accidentally have, I'd consider it to be working the way we'd want it to now

Reply:
Oh I'm with you! I think maybe the TWR overship having an "R" before an "F" just confused me, but sounds like the APP ownership is better now than it was before, and if `_F_` is on, TWR should already be on so it should never matter.

Thanks for the explanation!